gc.rb

```diff
@@ -30,12 +30,12 @@ module GC
   #  Note: These keyword arguments are implementation and version dependent. They
   #  are not guaranteed to be future-compatible, and may be ignored if the
   #  underlying implementation does not support them.
-  def self.start full_mark: true, immediate_mark: true, immediate_sweep: true
-    __builtin_gc_start_internal full_mark, immediate_mark, immediate_sweep
+  def self.start full_mark: true, immediate_mark: true, immediate_sweep: true, compact: false
```

Comment:
It'd be nice to have a NEWS entry for this

Reply:
Oh, I didn't think it's newsworthy. 😅 

I'll add a NEWS entry